src/Interceptor/Interceptor/CacheInterceptor.php

```diff
@@ -109,36 +89,69 @@ public function getSuffixPriority(): int
         return 20;
     }
 
-    private function buildCacheKey(Instance $instance, Cache $annotation): string
+    private function buildCacheKey(Instance $instance, Cache $attribute): string
     {
-        $version = $annotation->getVersion() !== null ? '.v' . $annotation->getVersion() : null;
+        $method = $instance->getMethod();
+
+        $parameters = $method->getParameters();
 
-        if ($annotation->getId() !== null) {
-            $parameters = $instance->getMethod()
-                ->getParameters();
+        $identifier = str_replace('\\', '.', $instance->getReflection()
+            ->getName()) . '.' . $method->getName();
 
-            return $this->resolveExpression($annotation->getId(), $parameters) . $version;
+        if (\count($parameters) > 0) {
+            foreach ($parameters as $parameterName => $parameterValue) {
+                $identifier .= '.' . $parameterName . '.' . md5(serialize($parameterValue));
```

Comment:
comment from notion 

Should use the fastest hash:
		- crc32c: 32 bits
		- xxh3 64 bits
		- xxh128: 128 bits
Benchmarks: https://php.watch/articles/php-hash-benchmark
Most recent and fastest: xxh3